### DISCUSSION TITLE
data about the hematopoiesis dataset

### DISCUSSION THREAD

**Discussion by tianchen2019** [GreenleafLab/ArchR#666] [Questions / Documentation]
I noticed that you built a the hematopoiesis dataset in the article.
Can you provide a way to download this data set?
I think this data set is very useful for us. Thanks.
![1](https://user-images.githubusercontent.com/46451175/112964623-15579b80-917b-11eb-9c3a-34edf88f69f8.png)

**Comment by rcorces** [comment]
I'm afraid that I dont have this in a readily shareable format. The links to download all of the respective publicly available datasets that went into this heme atlas are shown in one of the supplementary tables though.
